server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -103,6 +104,8 @@ export class ScriptoriumLambda implements IPartitionLambda {
                 // Duplicate key errors are ignored since a replay may cause us to insert twice into Mongo.
                 // All other errors result in a rejected promise.
                 if (error.code !== 11000) {
+                    this.context.log?.error(`Error inserting operation in the database: ${inspect(error)}`);
```

Comment:
@sharmakhushboo Since we only reject the promise if `error.code !== 11000`, I decided to log error only under the same circumstances. Let me know if you think we should log for every type of error.

Reply:
Can we also add an error log at the beginning of catch?

Reply:
Then I will just move this to outside the if statement, so it always logs.